imap_processing/codice/l0/decom_nhk.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+from space_packet_parser import parser, xtcedef
+
+from imap_processing import imap_module_directory
+
+# Define paths
+packet_file = Path("housekeeping_data.bin")
+xtce_document = Path(f"{imap_module_directory}/codice/packet_definitions/P_COD_NHK.xml")
+
+
+def decom_packets(packet_file: str, xtce_packet_definition: str):
```

Comment:
I don't think you need this function if you import the shared decom_packets function?

```python
from imap_processing.decom import decom_packets
```